docker-compose.production.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.4"
+
+services:
+  web:
+    build: .
+    ports:
+      - "8000:8000"
+    environment:
+      - "DJANGO_SETTINGS_MODULE=modernomad.settings.production"
+      - "DATABASE_URL=postgres://postgres@postgres/postgres"
+      - "SECRET_KEY=fdshjkfds"
```

Comment:
should this be here? Shouldn't this be set in Heroku.

Reply:
This isn't used on Heroku.

I'll document what this file is for - it is to run the app in a production environment locally to test the static files, etc. I have also switched Travis to use it for browser tests so the browser tests are run on something as close as possible to production.

Reply:
To me it would make sense to have a docker file that is used in production environment and if you wanted to run that locally to test things out you'd have to set any required environment variables yourself in your `.env` file for instance.

Reply:
Done!

Reply:
Or perhaps renaming this file to simulate-production or similar, just to be overly clear.

Reply:
No - that's not possible. There are various other runtime differences with the Docker setup that can't be done with environment variables (different command, code not mounted, no `client` service, etc).

Reply:
I understand the potential confusion, btu that seems a bit long-winded. I've added a comment to the file -- do you think that's sufficient?

Reply:
It'll do :)

Reply:
It can actually be used to deploy modernomad to production on services that support it, too. (Docker Enterprise, Kompose, etc.)